infra/nixos/concrexit.nix

```diff
@@ -312,6 +312,26 @@ in
                 alias = "${cfg.dir}/media/";
                 extraConfig = "internal;";
               };
+              locations."/apple-app-site-association" = {
+                alias = pkgs.writeText ''
+                  {
+                      "applinks": {
+                          "apps": [],
+                          "details": [
+                              {
+                                  "appID": "FRD6Y7E88Y.com.thaliapp",
+                                  "paths": [ "/pizzas/", "/events/*"]
```

Comment:
Maybe we want `/members/photos/` in here too. And in the future maybe also claiming sales orders, confirming payments, and the member list?

Reply:
I'm not sure what the app supports right now. @DeD1rk should things be added here?

Reply:
```
<data android:path="/pizzas/" />
<data android:path="/events/" />
<data android:pathPattern="/events/.*/" />
<data android:pathPattern="/members/photos/.*/" />
```

Reply:
Committees/societies/boards, members and sales will be supported in a while, but aren't yet. Ill make a PR when they are ready

Reply:
Now that I'm looking at it, apparently there's a new syntax for this file. The old version will still work but it might be good to upgrade as well.